scripts/actions/add-files-to-translation-queue.js

```diff
@@ -26,11 +41,8 @@ const translationDifference = (pendingFiles, prChanges) =>
       )
   );
 
-const slugIntersection = (pendingFiles, filesToRemove) =>
-  pendingFiles.filter((file) => filesToRemove.includes(file.slug));
-
-const humanTranslatedProjectID = process.env.HUMAN_TRANSLATION_PROJECT_ID;
-const machineTranslatedProjectID = process.env.MACHINE_TRANSLATION_PROJECT_ID;
+const humanTranslatedProjectID = 'HT_ID';
+const machineTranslatedProjectID = 'MT_ID';
```

Comment:
looks like we accidentally committed this. we will want to revert back to what we had before.

Reply:
oops